db/institutions.yml

```diff
@@ -262,4 +262,12 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+ufrankfurt:
+  aka: [University of Frankfurt]
+  city: Frankfurt
+  country: Germany
+  departments: {chemistry: {name: Department of Chemistry}, physics: {name: Department of Physics}}
```

Comment:
if you didn't find a department of physics, please don't add it.  Add Inst of theoretical physics, or whatever.